recipes/qxlsx/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package CXX)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+project(test_package LANGUAGES CXX)
 
 find_package(QXlsx REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} QXlsx::Core)
-set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+target_link_libraries(${PROJECT_NAME} PRIVATE QXlsx::Core)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
```

Comment:
In fact it requires C++ 17 only if Qt 6 is used. I could implement a logic in the validate method if qt 6 is used but:
- shouldn't C++ 17 requirement be handled by qt recipes ?
- since C++ requirement depends of the Qt version, is it ok to let `test_package/CMakeLists.txt` as is to handle both qt 5/6 testing ?

Reply:
I see, so it's a transitive requirement. Let's keep C++17 only here then.